Modules/zlibmodule.c

```diff
@@ -1958,12 +1988,17 @@ static PyType_Slot Decomptype_slots[] = {
 static PyType_Spec Decomptype_spec = {
     .name = "zlib.Decompress",
     .basicsize = sizeof(compobject),
-    .flags = Py_TPFLAGS_DEFAULT | Py_TPFLAGS_DISALLOW_INSTANTIATION,
+    .flags = (
+        Py_TPFLAGS_DEFAULT
+        | Py_TPFLAGS_DISALLOW_INSTANTIATION
```

Comment:
In a follow-up, I'll add the immutable flag as well. We still need the GC protocol as the type is not empty (I think? or maybe we don't need it, I'll need to check).